pkg/controller/plan/kubevirt.go

```diff
@@ -2058,6 +2083,33 @@ func (r *KubeVirt) libvirtDomain(vmCr *VirtualMachine, pvcs []*core.PersistentVo
 	return
 }
 
+func (r *KubeVirt) findConfigMapInNamespace(name string, namespace string) (configMap *core.ConfigMap, exists bool, err error) {
+	if err != nil {
+		return
+	}
+	list := &core.ConfigMapList{}
+	nameField := "metadata.name"
+	namespaceField := "metadata.namespace"
+	err = r.Destination.Client.List(
```

Comment:
do we expect more then one config map, maybe using `r.Destination.Client.Get` makes more sense here ?